### DISCUSSION TITLE
allure plugin required for bamboo 9.3

### DISCUSSION THREAD

**Discussion by manishsoni0407** [allure-framework/allure2#2163] [Questions & Support] [answered]
I have integrated my BDD framework with allure report however we use bamboo 9.3 and plugin is not available , Please send me the plugin for same.

**Comment by eroshenkoam** [comment, marked as answer]
We've fixed the plugin for Bamboo. Please check version 1.17.0. Let us know if everything works well, or if you encounter any issues.
https://marketplace.atlassian.com/apps/1217177/allure-report-for-bamboo/version-history#b1000160
